views/confirmation-still-required.html

```diff
@@ -38,8 +38,8 @@ <h1 class="govuk-panel__title">You've told us
         </p>
         {% endif %}
 
-        <p class="govuk-body">
-            <a href="print-confirmation" class="govuk-link">View printer-friendly version</a>
+        <p class="govuk-body hide-print-content print-link no-print" style="display: block;" aria-hidden="true" hidden="hidden">
+            <a href="javascript:if(window.print)window.print()" class="govuk-link">View printer-friendly version</a>
```

Comment:
what happens if JS is disabled?

Reply:
Then the link will be hidden and they will have to do it the usual way. They should have an email copy anyway. Same with screen readers.

Reply:
ok